pytaskflow/server/processor.py

```
@@ -30,6 +31,7 @@ def process(self):
             # 3. Set succeeded state
             succeeded_state = SucceededState(result=result, reason="Job performed successfully")
             self.storage.set_job_state(self.job.id, succeeded_state, expected_old_state=ProcessingState.NAME)
+            final_state = succeeded_state # Initialize final_state here for success path
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)